src/Shared/Navbar/NavbarDefault.jsx

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import {
+  Navbar,
+  MobileNav,
+  Typography,
+  Button,
+  IconButton,
+} from "@material-tailwind/react";
+import { FaHome, FaQuestionCircle, FaPhoneAlt } from "react-icons/fa";
+import { FaMagnifyingGlass } from "react-icons/fa6";
+import { MdFindInPage } from "react-icons/md";
+
+
+
+ 
+ function NavbarDefault() {
+  const [openNav, setOpenNav] = React.useState(false);
+  const user = true
+ 
+  React.useEffect(() => {
+    window.addEventListener(
+      "resize",
+      () => window.innerWidth >= 960 && setOpenNav(false),
+    );
+  }, []);
+ 
+  const navList = (
+    <ul className="mt-2 mb-4 flex flex-col gap-2 lg:mb-0 lg:mt-0 lg:flex-row lg:items-end lg:gap-6">
+      <Typography
+        as="li"
+        variant="small"
+        color="blue-gray"
+        className="flex items-center gap-x-2 p-1 font-medium"
+      >
+        <FaHome />
+
+ 
+        <a href="#" className="flex items-center">
+          Home
+        </a>
+      </Typography>
+
+      <Typography
+        as="li"
+        variant="small"
+        color="blue-gray"
+        className="flex items-center gap-x-2 p-1 font-medium"
+      >
+       <FaMagnifyingGlass />
+
+        <a href="#" className="flex items-center">
+          Lost
+        </a>
+      </Typography>
+
+      <Typography
+        as="li"
+        variant="small"
+        color="blue-gray"
+        className="flex items-center gap-x-2 p-1 font-medium"
+      >
+       <MdFindInPage />
+
+        <a href="#" className="flex items-center">
+          Found
+        </a>
+      </Typography>
+
+
+      <Typography
+        as="li"
+        variant="small"
+        color="blue-gray"
+        className="flex items-center gap-x-2 p-1 font-medium"
+      >
+      <FaQuestionCircle />
+
+
+        <a href="#" className="flex items-center">
+          How It Works
+        </a>
+      </Typography>
+
+      <Typography
+        as="li"
+        variant="small"
+        color="blue-gray"
+        className="flex items-center gap-x-2 p-1 font-medium"
+      >
+        <FaPhoneAlt />
+        <a href="#" className="flex items-center">
+          Contact Us
+        </a>
+      </Typography>
+    </ul>
+  );
+ 
+  return (
+    <Navbar className="mx-auto max-w-full px-4 py-2 lg:px-8 lg:py-4">
+      <div className=" mx-auto flex items-center justify-between text-blue-gray-900">
+        <Typography
+          as="a"
+          href="#"
+          className="mr-4 cursor-pointer py-1.5 font-medium text-xl"
+        >
+          <span className="text-red-500 text-2xl">L</span>ost<span className="text-red-500 text-2xl">N</span>foun<span className="text-red-500 text-2xl">D</span>
```

Comment:
If this whole block is a Title for designed lost and found, then it should be moved to separate component.